assume/reinforcement_learning/learning_utils.py

```diff
@@ -64,10 +69,18 @@ def __init__(self, action_dimension, mu=0.0, sigma=0.1, scale=1.0, dt=0.9998):
         self.dt = dt
 
     def noise(self):
-        noise = self.scale * np.random.normal(self.mu, self.sigma, self.act_dimension)
-        self.scale = self.dt * self.scale  # if self.scale >= 0.1 else self.scale
+        # TODO: document changes to normal action noise and different usage of dt parameter?
```

Comment:
I've changed the usage of noise_dt. Before, it was only possible to use it for exponential decay. Now, it can be set by the scheduling function independently (although currently only linear schedule implemented).

Where do I need to document those changes? And can I delete line 78, if it's documented and approved by everyone?